### ISSUE TITLE
Rotten tomatoes API has been locked down

### ISSUE THREAD

**Issue by jeanettehead** [mobilebridge/iosbridge-rottentomatoes#5]
It looks like they're not interested in being an API for free learning applications: https://developer.fandango.com/

**Comment by kmolo** [comment]
@jeanettehead good point.  Within Mobilebridge SF we have some existing API keys for the workshops we run.  We'll see what [CodePath](http://codepath.com) ends up using as a replacement demo app (we got our curricula from them), and probably migrate over to that.

Are you thinking of running a Mobilebridge workshop in Denver?  If so, we could work with you & share some of our API keys for your event.  Let us know.

**Comment by jeanettehead** [comment]
hey! it's like 6 months later, BUT we did run an iOSbridge workshop in Denver. 

We have a fork of this ( @sashei led the updating and the workshop ) that was updated to point to https://www.themoviedb.org/?language=en instead of rotten tomatoes. (https://github.com/denver-railsbridge/iosbridge-tmdb)

The workshop was super successful, and we're planning on running another one in May. Hopefully that'll be a good time to clean up the repo. 

We also created an installfest (a couple of the things in here are Denver specific, but mostly generic) which is here: http://denver-railsbridge.org/iosInstallfest/index.html

I have no idea how to get all of this content aligned and shared, do you have any recommendations? We can definitely work on our README as our next event gets closer, and try to organize all the resources we've built together.
